app/views/smart_answers/question.html.erb

```diff
@@ -37,7 +37,7 @@
 
         <%= render partial: "smart_answers/inputs/#{question.partial_template_name}", locals: { question: question } %>
 
-        <%= question.post_body %>
+        <%= content_tag(:div, question.post_body, class: "govuk-!-margin-bottom-4") if question.post_body.present? %>
```

Comment:
Without this change:

![next_step_without_div](https://user-images.githubusercontent.com/213040/98939592-94f11080-24e1-11eb-9958-af1e8404fd46.png)

After change

![next_step_with_div](https://user-images.githubusercontent.com/213040/98939620-9e7a7880-24e1-11eb-80e0-0e298409f2f7.png)

Reply:
Is this a bug introduced by the previous commit? or this a pre-existing issue?

Reply:
It's a pre-existing issue I believe. I spotted it when testing this code (it happens on the page immediately prior to the one this refactored code affects).

It needs fixing, but is so minor it seems silly to create a separate PR for it.